Kitodo/src/main/webapp/pages/Metadaten2rechts.xhtml

```diff
@@ -26,84 +25,86 @@
 
         <h:body style="margin: 0px 2px 2px 2px;" class="metadatenRechtsBody" onload="addableTypenAnzeigen();TreeReloaden()">
 
+            <!--TODO: Check this Script, it results in empty page with no error-->
```

Comment:
Please move this TODO line two lines down to the out commented part and adjust indent.